server/controllers/admin/placementRequests/withdrawalsController.ts

```diff
@@ -21,21 +23,31 @@ export default class WithdrawalsController {
         id: req.params.id,
         errors,
         errorSummary,
-        applicationId: placementRequest.applicationId,
+        expectedArrival: placementRequest.expectedArrival,
+        duration: placementRequest.duration,
       })
     }
   }
 
   create(): RequestHandler {
     return async (req: Request, res: Response) => {
       try {
-        if (!req.body.reason) {
+        const { reason, expectedArrival, duration } = req.body as {
+          reason: string | undefined
+          expectedArrival: string | undefined
+          duration: string
+        }
+
+        if (!reason) {
           throw new ErrorWithData({ 'invalid-params': [{ propertyName: `$.reason`, errorType: 'empty' }] })
         }
 
         await this.placementRequestService.withdraw(req.user.token, req.params.id, req.body.reason)
 
-        req.flash('success', 'Placement request withdrawn successfully')
+        req.flash(
+          'success',
+          `Placement request for ${placementLength(Number(duration))} starting on ${DateFormats.isoDateToUIDate(expectedArrival, { format: 'short' })} withdrawn successfully`,
```

Comment:
Might be better to have this in a util to keep logic out of the controller, but this is fine for now

Reply:
Good idea, I'm working on doing the same thing for placement applications in a subsequent PR so will fix there